modules/gapi/test/infer/gapi_infer_onnx_test.cpp

```diff
@@ -164,7 +172,7 @@ void remapYolo(const std::unordered_map<std::string, cv::Mat> &onnx,
     // Configured output
     cv::Mat& out = gapi.begin()->second;
     // Simple copy
-    copyToOut<float>(in, out);
+    copyToOut<float>(in, -1.f, out);
```

Comment:
Where does this magic number come from?

Reply:
From user. It is end of data mark. Output has positive values and this mark separate correct data and garbage. `copyToOut` function is part of remapping function.